### ISSUE TITLE
docs: propose to use code shell style for cli command

### ISSUE THREAD

**Issue by alan-agius4** [angular/angular#26574]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

![image](https://user-images.githubusercontent.com/17563226/47205738-0ef84f00-d387-11e8-889e-762cbd29e74b.png)

## Expected behavior

![image](https://user-images.githubusercontent.com/17563226/47205723-043dba00-d387-11e8-8684-5d0811378d21.png)

I propose that the CLI commands are displayed in the shell like code example box. To be consistent with other shell commands in the documentation, and also they are visibility.

This is relatively simple to do, and I am open to do it if the editor agrees.

**Comment by jenniferfell** [comment]
Interesting idea. @jbogarthyde @petebacondarwin ? 

Related: The [Google Dev Doc Style Guide](https://developers.google.com/style/code-syntax) calls for including the command prompt, but I think that only makes sense in guides where we show what users enter and what the command writes back. We're not using command prompts at all in doc command line examples.

**Comment by jenniferfell** [comment]
For fun I made this my survey of the week for GDEs. They've been a pretty easy way to get immediate feedback.

**Comment by Splaktar** [comment]
I'm on the fence about this one.

Why does the syntax highlighting vary across `ng add` and `ng config`? Blue in `add` and black in `config`.
![screen shot 2018-10-19 at 2 01 36 pm](https://user-images.githubusercontent.com/3506071/47235611-92c23380-d3a7-11e8-9f4f-a38365c23f46.png)

We lose the syntax coloring in the console view, but if it's inconsistent, then maybe that's a good thing?

**Comment by gkalpak** [comment]
WRT different color, see #26976.

**Comment by sjtrimble** [comment]
@gkalpak what does #26976 have to do with this?

I think we should definitely maintain the CLI styling (green text on black background) to be consistent - and also to draw attention to that command text on the page.

Is there any reason not to do this?

**Comment by gkalpak** [comment]
> @gkalpak what does #26976 have to do with this?

Sorry, wrong link. I meant #26570. It answers to @Splaktar's question about why the syntax highlighting varies across `ng add` and `ng config` (which is not related to the rest of the discussion).

##
Regarding the styling:

I am a little torn. The green on black usually indicates a command that the user can copy/paste into a terminal and run, which is not the case here. But the current styling hardly distinguishes the command from regular text.

BTW, this is also an issue with function API docs (e.g. [animations/animate][1]:

![api-function](https://user-images.githubusercontent.com/8604205/59388493-8a729a80-8d74-11e9-98a0-ebe06c45549d.png)

##
If we could come up with a better, more distinguishable styling (that does not look like a terminal), I would prefer that. Maybe something similar to class APIs (e.g. [animations/AnimationBuilder][2]):

![api-class](https://user-images.githubusercontent.com/8604205/59388614-ea694100-8d74-11e9-8e7f-07224a355b0b.png)

[1]: https://angular.io/api/animations/animate
[2]: https://angular.io/api/animations/AnimationBuilder

**Comment by sjtrimble** [comment]
I kind of like what Google Cloud does where the styles are the same for instructional versus ones you can copy past directly into the terminal (https://cloud.google.com/deployment-manager/docs/quickstart). They don't have the green/black color palette we have, however.

How do we feel about having the existing boxed styles for both function API docs and the terminal -> same style, just remove the copy icon when it's not intended to be copied and pasted?

I'm big on consistency and patterns. In my mind we either get rid of the black/green styles and distinguish terminal code examples with the `$` before it, OR we keep the black/green styles for terminal and use that for the CLI docs as well (without copy icon as suggested above).

Another example where they use their regular in-paragraph code styling even if it's terminal commands when speaking about them: https://docs.globus.org/cli/high-assurance/

**Comment by alan-agius4** [comment]
> How do we feel about having the existing boxed styles for both function API docs and the terminal -> same style, just remove the copy icon when it's not intended to be copied and pasted?

I personally like this idea, but what I am really after is consistence mostly.

**Comment by gkalpak** [comment]
@sjtrimble, I don't understand your examples. I only see commands that are supposed to be typed in the terminal there 😕
We have two different things here:
- Showing a command as it should be typed in the terminal.
- Showing the "call signature" for a command (i.e. what arguments it accepts, which are optional, etc.).

I don't feel these two things should have the same style (and I am big on consistency too btw 😛).
If anything, the "call signature" is more similar in nature with showing classes or function signatures, so I would consider making them look the same (as suggested in https://github.com/angular/angular/issues/26574#issuecomment-501467644).

But, I don't feel too strongly about it. Leaving it up to you 😎

**Comment by ajitsinghkaler** [comment]
@gkalpak I think this has been added and we can close this issue.

**Comment by gkalpak** [comment]
No, this has not been addressed yet. (We haven't decided if/how it should be addressed :grin:)

**Comment by petebacondarwin** [comment]
Let's just stick a box round this bit of text to make it stand out from the surrounding text, but not change the theming to bash style.  Sound OK @alan-agius4 ?
